src/Html/Events/Extra.elm

```diff
@@ -235,34 +222,25 @@ targetSelectedIndex =
 -}
 onClickPreventDefault : msg -> Attribute msg
 onClickPreventDefault msg =
-    onWithOptions "click"
-        { defaultOptions | preventDefault = True }
-        (Json.succeed msg)
+    custom "click" (Json.succeed { message = msg, stopPropagation = False, preventDefault = True })
```

Comment:
From reading the elm/virtual-dom source code, I believe there might be subtle differences between this and using `Html.Events.preventDefaultOn`,  and I think we should probably be using the latter here.